experiments/AtmosGCM/heldsuarez.jl

```diff
@@ -1,121 +1,137 @@
 #!/usr/bin/env julia --project
 using ClimateMachine
-using ArgParse
-
-s = ArgParseSettings()
-@add_arg_table! s begin
-    "--number-of-tracers"
-    help = "Number of dummy tracers"
-    metavar = "<number>"
-    arg_type = Int
-    default = 0
-end
-
-parsed_args = ClimateMachine.init(parse_clargs = true, custom_clargs = s)
-const number_of_tracers = parsed_args["number-of-tracers"]
+ClimateMachine.init(parse_clargs = true)
 
 using ClimateMachine.Atmos
 using ClimateMachine.Orientations
 using ClimateMachine.ConfigTypes
 using ClimateMachine.Diagnostics
 using ClimateMachine.GenericCallbacks
 using ClimateMachine.ODESolvers
+using ClimateMachine.TurbulenceClosures
 using ClimateMachine.SystemSolvers: ManyColumnLU
 using ClimateMachine.Mesh.Filters
 using ClimateMachine.Mesh.Grids
 using ClimateMachine.TemperatureProfiles
 using ClimateMachine.Thermodynamics:
-    air_temperature, internal_energy, air_pressure
-using ClimateMachine.TurbulenceClosures
+    air_pressure, air_density, air_temperature, total_energy, internal_energy
 using ClimateMachine.VariableTemplates
 
-using Distributions: Uniform
 using LinearAlgebra
 using StaticArrays
-using Random: rand
 using Test
 
 using CLIMAParameters
-using CLIMAParameters.Planet: R_d, day, grav, cp_d, cv_d, planet_radius
+using CLIMAParameters.Planet:
+    MSLP, R_d, day, cp_d, cv_d, grav, Omega, planet_radius
 struct EarthParameterSet <: AbstractEarthParameterSet end
 const param_set = EarthParameterSet()
 
-struct HeldSuarezDataConfig{FT}
-    T_ref::FT
-end
+import CLIMAParameters
+#CLIMAParameters.Planet.planet_radius(::EarthParameterSet) = 6.371e6 / 125.0
 
-function init_heldsuarez!(bl, state, aux, coords, t)
+function init_baroclinic_wave!(bl, state, aux, coords, t)
```

Comment:
Why initialize HS with this detailed a baroclinic wave?

Reply:
That's a mistake. Needs to be reverted.